pkg/controller/volume/attachdetach/metrics/metrics.go

```diff
@@ -44,22 +44,50 @@ const (
 )
 
 var (
-	inUseVolumeMetricDesc = metrics.NewDesc(
+	// Deprecated: inUseVolumeMetricDescDeprecated is deprecated,
+	// and will be removed at v1.33. Please use inUseVolumeMetricDesc instead.
+	inUseVolumeMetricDescDeprecated = metrics.NewDesc(
 		metrics.BuildFQName("", "storage_count", "attachable_volumes_in_use"),
 		"Measure number of volumes in use",
 		[]string{"node", "volume_plugin"}, nil,
 		metrics.ALPHA, "")
 
-	totalVolumesMetricDesc = metrics.NewDesc(
+	// Deprecated: totalVolumesMetricDescDeprecated is deprecated,
+	// and will be removed at v1.33. Please use totalVolumesMetricDesc instead.
+	totalVolumesMetricDescDeprecated = metrics.NewDesc(
 		metrics.BuildFQName("", "attachdetach_controller", "total_volumes"),
 		"Number of volumes in A/D Controller",
 		[]string{"plugin_name", "state"}, nil,
+		metrics.ALPHA, "1.33.0")
+
+	// Deprecated: ForceDetachMetricCounterDeprecated is deprecated,
+	// and will be removed at v1.33. Please use ForceDetachMetricCounter instead.
+	ForceDetachMetricCounterDeprecated = metrics.NewCounterVec(
```

Comment:
The comment is invisible to end users, it may be better to include it in the help text. What do you think?

By the way, the PR should be merged in 1.33 due to the code freeze, so we can not remove the old metrics in the same release. Considering compatibility, we should give the users more time to migrate to the new metrics.